platforms/jvm/code-quality/src/main/groovy/org/gradle/api/plugins/quality/Pmd.java

```diff
@@ -339,19 +293,8 @@ public Property<Integer> getRulesMinimumPriority() {
      * @since 2.1
      */
     @Input
-    @ToBeReplacedByLazyProperty
-    public boolean isConsoleOutput() {
-        return consoleOutput;
-    }
-
-    /**
-     * Whether or not to write PMD results to {@code System.out}.
-     *
-     * @since 2.1
-     */
-    public void setConsoleOutput(boolean consoleOutput) {
-        this.consoleOutput = consoleOutput;
-    }
+    @ReplacesEagerProperty(originalType = boolean.class)
+    public abstract Property<Boolean> getConsoleOutput();
```

Comment:
Do we replace references to `isConsoleOutput = true` in Kotlin with `getConsoleOutput().set(true)`?

Reply:
For binary compatibility we replace such references. For source compatibility we are adding `getIsX` getter.
I actually missed that in that case, so good catch, added:
```
public Property<Boolean> getIsConsoleOutput() {
```